CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
```

Comment:
please fix this. Add more detials here, new version will be 0.34, new FDI is 1.17